### ISSUE TITLE
What are PHP developers supposed to use now that ApplicationInsights-PHP has been listed as abandoned and archived?

### ISSUE THREAD

**Issue by JimmyBanks** [microsoft/ApplicationInsights-Announcements#31]
Please advise.

**Comment by DawgFan** [comment]
Hi Jimmy, Unfortunately we do not currently have a supported SDK for PHP, but we are working on providing a solution via OpenTelemetry. Sign-up for our early adopters community at https://aka.ms/AzMonOT/.

**Comment by itsfadnis** [comment]
Any update on this?

**Comment by mattmccleary** [comment]
Hi there, we are testing a supported OTLP path to Azure Monitor internally right now. No plans to release it for Azure Monitor customers in 2021. The best way to make your voice known is to add your scenario at https://aka.ms/AzMonOT/. In the meantime, some customers have had success using the [OpenTelemetry-Collector Azure Monitor exporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/azuremonitorexporter) (which is not officially supported by Microsoft right now) for langauges such as PHP and Go.

**Comment by akserikawa** [comment]
Are there any official support plans for PHP in 2022-2023?

**Comment by yingqunpku** [comment]
> 

Yes, we plan to support languages with OTel SDKs in the near future. Stay tuned
And may I know what telemetry type support you're expecting (the most)? Logs, traces, or metrics, or all of them?

**Comment by CxDevLead** [comment]
Three years with an unsupported library on the most popular web development language is unacceptable.

**Comment by akserikawa** [comment]
> > 
> 
> Yes, we plan to support languages with OTel SDKs in 2023. Stay tuned And may I know what telemetry type support you're expecting (the most)? Logs, traces, or metrics, or all of them?

I would like to see PHP being supported with as much features as possible. Currently deploying PHP applications on Azure without an included APM solution feels incomplete.

**Comment by vwatel** [comment]
Any update over here using Opentelemetry with PHP for Application Insights?
@yingqunpku do you have some information on timeline when in 2023 this will be ready?

**Comment by yingqunpku** [comment]
@vwatel Sorry, the timeline is still TBD, and the 2023 estimation is also likely to change, so I have updated my reply above.

**Comment by vwatel** [comment]
Thank you for the update @yingqunpku.
I have also asked Microsoft contacts meanwhile, and the answer was clearly there is no timeline supporting OTL with PHP, and might not even be supported at all.

**Comment by micaelvitor** [comment]
ZzZZzzZ

**Comment by CamiloTerevinto** [comment]
Any updates on this? @yingqunpku @mattmccleary
